src/search/landmarks/landmark_count_heuristic.cc

```diff
@@ -263,20 +254,44 @@ bool LandmarkCountHeuristic::generate_helpful_actions(
 }
```

Comment:
github doesn't let me comment on it directly because it's not close enough to a modified line, so I'm adding this comment here: The two lines with the `return false` case above this block no longer make sense and should be removed. There is no need to add special code to avoid iterating over an empty vector.